stdpopsim/catalog/GasAcu/species.py

```diff
@@ -116,7 +116,7 @@
 
 _species = stdpopsim.Species(
     id="GasAcu",
-    ensembl_id="9307941",
+    ensembl_id="gasterosteus_aculeatus",
```

Comment:
perhaps this is why the assembly dissappeared?

Reply:
no, the original ensembl_id was incompatible with the REST API and the maintenance script...